polkadot/node/subsystem-types/src/messages.rs

```diff
@@ -147,8 +147,8 @@ pub enum CandidateValidationMessage {
 		CandidateReceipt,
 		Arc<PoV>,
 		ExecutorParams,
-		/// Execution timeout
-		PvfExecTimeoutKind,
+		/// Execution timeout and retry
```

Comment:
Maybe this would be more clear:

```suggestion
		/// Execution kind, used for timeouts and retries
```

And below also.